frontend/src/components/modals/dropdown/DropdownMenu.tsx

```diff
@@ -1,37 +1,51 @@
 import {
-	DropdownWrapper,
-	DropdownList,
-	DropdownListItem
+    DropdownWrapper,
+    DropdownList,
+    DropdownListItem
 } from "./DropdownMenu.styled";
 import useDropdownMenu from "./useDropdownMenu";
+import { useNavigate } from "react-router-dom";
+import useToken from "../../../hooks/useToken";
 
 type IDropdownMenu = {
-	showModal: (modalName: string) => void;
-	changeMenuVisible: () => void;
+    showModal: (modalName: string) => void;
+    changeMenuVisible: () => void;
 };
 
 const DropdownMenu = ({ showModal, changeMenuVisible }: IDropdownMenu) => {
-	const { ref } = useDropdownMenu(changeMenuVisible);
-
-	return (
-		<>
-			<DropdownWrapper>
-				<DropdownList ref={ref}>
-					<DropdownListItem onClick={() => showModal("edit")}>
-						Edytuj konto
-					</DropdownListItem>
-
-					<DropdownListItem onClick={() => showModal("deleteAvatar")}>
-						Usuń Avatar
-					</DropdownListItem>
-
-					<DropdownListItem onClick={() => showModal("deleteAccount")}>
-						Usuń konto
-					</DropdownListItem>
-				</DropdownList>
-			</DropdownWrapper>
-		</>
-	);
+    const { ref } = useDropdownMenu(changeMenuVisible);
+    const navigate = useNavigate();
+    const { setAccessToken } = useToken();
+
+    const handleLogout = () => {
```

Comment:
i think, this should be inside useDropdownMenu, cause we follow this strategy, for create all logic and handlers inside custom hook